### DISCUSSION TITLE
Make several cuts in the same video

### DISCUSSION THREAD

**Discussion by mamomo** [mifi/lossless-cut#1800] [Support]
Surely it can be done, but I don't know how to trim multiple parts of the video and then make it a single video with those parts trimmed.

**Comment by mifi** [comment]
use the <kbd>+</kbd> to add a segment

**Comment by glenpinn** [comment]
I know this is an old Topic, but i have done this several times now using my 4k/60p HEVC video files from my camera, in some videos i have cut over 15 segments and merged the remaining parts back to a single file, and it works quicker than i can bling, no loss of quality, no format change, my output file is identical to my source file, but shorter.
